pgmult/lda.py

```diff
@@ -391,9 +413,20 @@ def __init__(self, data, timestamps, K, alpha_theta):
     def initialize_parameters(self):
         self.sigmasq_states = 0.1  # TODO make this learned, init from hypers
 
-        mean_psi = compute_uniform_mean_psi(self.V)[0][None,:,None]
-        self.psi = np.tile(mean_psi, (self.T, 1, self.K))
-
+        # finale: added this to the initialization (must already be
+        # correct size, do that in the top level file or as another
+        # function in the top level file), not here
+        #
+        # Melih: pi_to_psi expects rows to be normalized
+        # and only takes arrays of size <= 2.
+        # Need confirmation on whether double transpose below
+        # results in desired behavior.
+        if self.param_set['use_init_beta']:
+            self.psi = np.array(map(lambda x: pi_to_psi(x.T).T, self.param_set['init_beta']))
```

Comment:
@finaledoshivelez modified assignment to psi. Results:
![dtm_from_lda](https://cloud.githubusercontent.com/assets/1884921/8934613/804e7dd6-3516-11e5-98cf-ad91933d8467.png)